kernel/comps/logger/src/aster_logger.rs

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use log::{Metadata, Record};
+use ostd::timer::Jiffies;
+use spin::Once;
+
+/// The logger used for Asterinas.
+pub(super) struct AsterLogger {}
+
+pub(super) static LOGGER: Once<AsterLogger> = Once::new();
+
+#[cfg(feature = "log_color")]
+fn print_logs(record: &Record, timestamp: f64) {
+    use owo_colors::Style;
+
+    let timestamp_style = Style::new().green();
+    let record_style = Style::new().default_color();
+    let level_style = match record.level() {
+        log::Level::Error => Style::new().red(),
+        log::Level::Warn => Style::new().bright_yellow(),
+        log::Level::Info => Style::new().blue(),
+        log::Level::Debug => Style::new().bright_green(),
+        log::Level::Trace => Style::new().bright_black(),
+    };
+
+    super::_print(format_args!(
+        "{} {:<5}: {}\n",
+        timestamp_style.style(format_args!("[{:>10.3}]", timestamp)),
+        level_style.style(record.level()),
+        record_style.style(record.args())
+    ));
+}
+
+#[cfg(not(feature = "log_color"))]
+fn print_logs(record: &Record, timestamp: f64) {
+    super::_print(format_args!(
+        "{} {:<5}: {}\n",
+        format_args!("[{:>10.3}]", timestamp),
+        record.level(),
+        record.args()
+    ));
+}
```

Comment:
For code readability, I think `print_logs` should be moved down below the `log::Log` implementation.

This is because the trait implementation is more important, and for a newcomer, they need to read the trait implementation first, then check out the details of the `print_logs` implementation.